edifact-edi/2.7/modules/ROOT/pages/edifact-edi-connector-reference.adoc

```diff
@@ -8,194 +8,391 @@
 |===
 
-=== Output
+==== Output
 
 [cols="30a,70a"]
 |===
 | Type | Object
 |===
 
-=== For Configurations
+==== For Configurations
 
 * config
 
-=== Throws
+==== Throws
 
 * EDIFACT:SCHEMA
 * EDIFACT:WRITE
 * EDIFACT:UNKNOWN
 * EDIFACT:PARSE
 
-== Write
+[[write]]
+=== Write
 
 `<edifact:write>`
 
-Transforms the maps and lists structure of EDI data to EDI text stream.
+Transforms the maps and lists the structure of EDI data in the EDI text stream.
 
-=== Parameters
+==== Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | |x
-| Write Content | Object |  |  `#[payload]` |
-| Streaming Strategy | repeatable-in-memory-stream
-repeatable-file-store-stream
-non-repeatable-stream |  Configure to use repeatable streams and their behavior. |  |
-| Target Variable | String |  The name of a variable on which the operation's output is placed. |  |
-| Target Value | String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
+| Write Content | Object | Output payload. |  `#[payload]` |
+| Streaming Strategy a| Enumeration, one of:
+
+* <<repeatable-file-store-stream>>
+* <<repeatable-in-memory-stream>>
+* Non repeatable stream |  Configures how Mule processes streams. | Repeatable file store stream |
+| Target Variable | String |  Name of the variable that stores the operation's output. |  |
+| Target Value | String |  Expression that evaluates the operation’s output. The outcome of the expression is stored in the *Target Variable* field. |  `#[payload]` |
 |===
 
-=== Output
+==== Output
 
 [cols="30a,70a"]
 |===
 | Type | Binary
 |===
 
-=== For Configurations
+==== For Configurations
 
 * config
 
-=== Throws
+==== Throws
 
 * EDIFACT:SCHEMA
 * EDIFACT:WRITE
 * EDIFACT:UNKNOWN
 * EDIFACT:PARSE
 
-== Write batch
+[[write-batch]]
+=== Write batch
 
 `<edifact:write-batch>`
 
-Merge individual EDIFACT messages created and accumulated over a period of time into a single interchange (UNB).
+Merges individual EDIFACT messages created and accumulated over a period of time into a single interchange (UNB).
 
-=== Parameters
+==== Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Write Content | Object |  |  `#[payload]` |
-| Target Variable | String |  The name of a variable on which the operation's output is placed. |  |
-| Target Value | String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
+| Configuration | String | Name of the configuration to use. | |x
+| Write Content | Object | Output payload. |  `#[payload]` |
```

Comment:
```suggestion
| Batch Content | Binary | Input payload. |  `#[payload]` |
```
I noticed that it has the same value as the write operation, however, it has another name and it is a combination of the read and write operations. The input payload is an input stream (binary) and the output is also binary.